types/index.d.ts

```diff
@@ -0,0 +1,10 @@
+export * from "../lib/build/types";
```

Comment:
Also, edit playground app in the test folder (it's called with-typescript) to make use of these types for all recipes and check that it's working as expected.

Reply:
done